types/pdfkit/index.d.ts

```diff
@@ -126,9 +126,9 @@ declare namespace PDFKit.Mixins {
     type PDFFontSource = string | Buffer | Uint8Array | ArrayBuffer;
 
     interface PDFFont {
-        font(buffer: Buffer): this;
-        font(src: string, size?: number): this;
-        font(src: string, family?: string, size?: number): this;
```

Comment:
Probably, we should just use the PDFFontSource type as type for the src argument.
That would match exactly all the types that are supported by [PDFFontFactory](https://github.com/foliojs/pdfkit/blob/3f69586f3cb9d46cb58f255920863bbcbc5f2acd/lib/font_factory.js).

Reply:
I agree. Also, `size` will also work when src is any `PDFFontSource` variant.
Same reasoning goes for `family`: it doesn't matter what the runtime type for buffer/src is.

This reduces an overload.